evalai/urls.py

```diff
@@ -22,24 +22,24 @@
 from django.views.generic.base import TemplateView
 from drf_yasg import openapi
```

Comment:
Should we remove this and all its related components since we are adding a new package?